README.md

```diff
@@ -322,6 +322,62 @@ Class['apt::update'] ->
 Class['::mysql::client']
 ```
 
+### Install MySQL 5.6 server on CentOS
+
+This minimal example shows how to install MySQL Community Server 5.6 on Centos 7.3 using Puppet 3.8.7 using hiera with puppetlabs-mysql=3.9.0
```

Comment:
That makes sense. I think the version numbers make the sentence harder to parse, so I'd suggest something like:

You can install MySQL Community Server on CentOS using the mysql module and Hiera. This example was tested with the following versions:

* MySQL Community Server 5.6
* CentOS 7.3
* Puppet 3.8.7
* puppetlabs-mysql 3.9.0